packages/playwright/src/matchers/expect.ts

```diff
@@ -354,9 +355,9 @@ class ExpectMetaInfoProxyHandler implements ProxyHandler<any> {
       // toPass and poll matchers can contain other steps, expects and API calls,
       // so they behave like a retriable step.
       const stepInfo = {
-        category: (matcherName === 'toPass' || this._info.poll) ? 'step' : 'expect',
+        category: (matcherName === 'toPass' || this._info.poll) ? 'test.step' : 'expect' as TestStepCategory,
         apiName,
-        title: trimLongString(title, 1024),
```

Comment:
Why did it go away?

Reply:
It was a wrong place for it - it was lossy.